plugins/topology/src/hooks/usePodLogs.ts

```diff
@@ -3,7 +3,7 @@ import useAsyncRetry from 'react-use/lib/useAsyncRetry';
 import useInterval from 'react-use/lib/useInterval';
 
 import { useApi } from '@backstage/core-plugin-api';
-import { kubernetesProxyApiRef } from '@backstage/plugin-kubernetes';
+import { kubernetesProxyApiRef } from '@backstage/plugin-kubernetes-react';
```

Comment:
Followup to my previous comment, Replace all these `kubernetesProxyApiRef` to point to the plugin's version of `kubernetesProxyApiRef`.